jobfunnel/tests/settings/settings1.yaml

```diff
@@ -0,0 +1,14 @@
+# paths
+output_path: 'fish'
```

Comment:
Perhaps we can either make this during the test (within a temporary file) - otherwise we should rename this to something like test.yaml to prevent confusion for new users looking at existing settings.yaml.

Reply:
Setting files in the for tests have been removed and been replaced by temporary yaml files created from a config dictionary. Good suggestion!